src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2378,116 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
     )
 
 
+@run_spec_function("cleva")
+def get_cleva_spec(task: str, version: str, subtask: str = None, prompt_id: int = 0) -> RunSpec:
+    CLEVAScenario.download_dataset(version)
```

Comment:
Add `from .scenarios.cleva_scenario import CLEVAScenario  # noqa` as the first line of this method.